.gitmodules

```diff
@@ -1,9 +1,3 @@
-[submodule "subprojects/wf-config"]
```

Comment:
-1 for this change, I believe it was unintentional

Reply:
Yes, it was

Reply:
I reverted this; I don't understand that it still shows those modules removed (I have them in a separate directory).

Reply:
Should be fixed